scss/modules/_nav.scss

```diff
@@ -204,7 +204,7 @@
     .site-nav__link {
 
       &.is-parent .site-nav__link__title {
-        border-bottom-color: $secondary;
+        border-bottom-color: $deep-red;
```

Comment:
In this case it doesn't make a difference, but good to be consistent. Just updated.